crates/pallets/starknet/src/lib.rs

```diff
@@ -815,8 +879,12 @@ impl<T: Config> Pallet<T> {
         let provided_seq_addr = Self::sequencer_address();
         let sequencer_address = ContractAddress(starknet_api::api_core::PatriciaKey(StarkFelt(provided_seq_addr)));
         let vm_resource_fee_cost = HashMap::default();
-        // FIXME: https://github.com/keep-starknet-strange/madara/issues/329
-        let gas_price = 10;
+
+        let gas_price = match Self::gas_price_l1() {
+            0 => 10, // default gas price
```

Comment:
10 was just a random value you can use unwrap_or_default here imo

Reply:
But won't that set the value to be 0 always? Or there's way to set custom default value?

Reply:
nah it would set it to 0 but that's not a problem for now

Reply:
so should change the value in tests accordingly?

Reply:
yes plz